packages/react-next/src/components/SwatchColorPicker/SwatchColorPicker.types.ts

```diff
@@ -45,24 +46,30 @@ export interface ISwatchColorPickerProps {
    */
   colorCells: IColorCellProps[];
 
+  /**
+   * @deprecated Provide `selectedId` if controlled or `defaultSelectedId` if uncontrolled
+   */
+  isControlled?: boolean;
+
   /**
    * Indicates whether the SwatchColorPicker is fully controlled.
    * When true, the component will not set its internal state to track the selected color.
    * Instead, the parent component will be responsible for handling state in the callbacks like
    * `onColorChanged`.
-   *
-   * NOTE: This property is a temporary workaround to force the component to be fully controllable
-   * without breaking existing behavior
    */
```

Comment:
This comment needs to be updated to something like the following:
```
  /**
   * ID of the default selected color swatch. Only provide this if the SwatchColorPicker is an
   * uncontrolled component; otherwise, use the `selectedId` property.
   */
```